site/src/docs/components/login/index.mdx

```diff
@@ -154,17 +154,43 @@ Detailed information is in the <UsagePageAnchorLink anchor="signals">signals sec
 ### Common settings for localhost
 
 Every service must have its own `client_id` and `scope`, but these settings can be used for testing.
-They only work in `http://localhost:3000`. Make sure you have `/callback` and `/silent_renew.html` configured.
+They only work in `http://localhost:3000`. Make sure you have `redirect_uri`, `silent_redirect_uri` and `post_logout_redirect_uri` registered at the OIDC server.
 
 ```js
 const loginProviderProps: LoginProviderProps = {
   userManagerSettings: {
     authority: 'https://tunnistamo.dev.hel.ninja/',
     client_id: 'exampleapp-ui-dev',
     scope: 'openid profile email https://api.hel.fi/auth/helsinkiprofiledev https://api.hel.fi/auth/exampleappdev',
-    redirect_uri: `${window.origin}/callback`,
+    redirect_uri: `YOUR_REGISTERED_REDIRECT_URL`,
+    silent_redirect_uri: `URL_TO_YOUR_REGISTERED_LOCAL_SILENT_RENEW.HTML`,
+    post_logout_redirect_uri: `YOUR_REGISTERED_LOGOUT_URL`,
   },
   apiTokensClientSettings: { url: 'https://tunnistamo.dev.hel.ninja/api-tokens/' },
   sessionPollerSettings: { pollIntervalInMs: 10000 },
 };
 ```
+
+If you are using Keycloak, then use these settings:
```

Comment:
If Keycloak will be replacing Tunnistamo, should we add a notification about that somewhere in docs? Maybe in Index page, Introduction section?